addons/common/functions/fnc_drawHintLineIcon.sqf

```diff
@@ -0,0 +1,61 @@
+#include "script_component.hpp"
```

Comment:
This function seems a bit too specific for these keybinds. I like the idea of common functions that allow for easy 3D hints. I think this should be split into 2 functions, one for line drawing and the other for icon drawing (`drawHintLine`, `drawHintIcon`).

Also, add support for objects as positions so lines and icons update as objects move.

Reply:
How does this look? 8a91f43